app/components/TrackComponent/index.js

```diff
@@ -67,9 +93,23 @@ const ButtonLabel = styled.span`
   }
 `;
 
-export function TrackComponent({ collectionName, artistName, imageUrl, trackName, trackUrl, handlePauseTrackWrapper }) {
+export function TrackComponent({ singleTrackData, isShowDetailsButton, isShowDetails, handlePauseTrackWrapper }) {
   const [isTrackPlaying, setIsTrackPlaying] = useState(false);
+  const {
+    artistName,
+    artworkUrl100: imageUrl,
+    previewUrl,
+    trackTimeMillis: trackDuration,
+    collectionName,
+    trackName,
+    trackId,
+    country,
+    primaryGenreName,
+    kind,
+    wrapperType
+  } = singleTrackData || {};
```

Comment:
There is something called defaultProps that you can use to set the default data. use that. Lets keep it trackData instead of singleTrackData

Reply:
defaultProps was not working for `artistName`, I tried using defaultProps first only. Here's the screenshot.
<img width="619" alt="image" src="https://user-images.githubusercontent.com/114564643/195008817-7220e58a-c275-457e-8fb5-240caba9357e.png">